com.bitwig.BitwigStudio.yaml

```diff
@@ -0,0 +1,78 @@
+---
+app-id: com.bitwig.BitwigStudio
+command: bitwig-studio
+runtime-version: "20.08"
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+rename-desktop-file: bitwig-studio.desktop
+rename-icon: bitwig-studio
+finish-args:
+  # TODO: we need MIDI, Audio, DRI, and realtime performances
+  - --device=all
+  - --nodevice=kvm
+
+  # pipewire
+  - --filesystem=xdg-run/pipewire-0
+  - --socket=system-bus
```

Comment:
I'm pretty sure the whole system bus is not needed. Use `flatpak run --log-system-bus ...` to find out what exact services are needed (look for "Filtering message ..." lines) and add them as `--system-talk-name=...`

Reply:
I check it.

Reply:
Indeed the system-talk-name was not necessary. I removed it.

Reply:
Actually, this is usually necessary (but not always) for jack + pipewire support for rt-kit.

symptoms include message about not being able to access the bus (in the console), or even crashes (some client app don't react well).

Reply:
Does rt-kit even work for flatpak apps anyway? AFAIK the client is supposed to send its PID, which won't match inside and outside the sandbox.
Anyway, I guess we can add `--system-talk-name=org.freedesktop.RealtimeKit1` just in case.

Reply:
this doesn't help either.